mkdocs/tests/structure/page_tests.py

```diff
@@ -366,7 +366,24 @@ def test_page_title_from_markdown_strip_formatting(self, docs_dir):
         pg = Page(None, fl, cfg)
         pg.read_source(cfg)
         pg.render(cfg, fl)
-        self.assertEqual(pg.title, '*Hello &mdash; beautiful world')
+        self.assertEqual(pg.title, '*Hello — beautiful world')
```

Comment:
I'm not convinced this behavior is desired here. I expect we do not want HTML entities to be unescaped. If so, we will need to clone the striptags function and eliminate the call to `unescape`. See [this comment](https://github.com/mkdocs/mkdocs/issues/3357#issuecomment-1887767130) for more.